### PR TITLE
update hpa test file

### PR DESCRIPTION
**Pull request by yapei**

### CODE REVIEW

File: testdata/hpa/hpa_rubyex.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-apiVersion: autoscaling/v2beta1
+apiVersion: autoscaling/v2beta2
```
Comment (liangxia) [inline testdata/hpa/hpa_rubyex.yaml:1]: Will it works on both 4.12 and 4.6 ?

Comment (yapei) [reply testdata/hpa/hpa_rubyex.yaml:1]: I just ran regression against 4.9, let me build a 4.6 cluster and let's see

Comment (yapei) [reply testdata/hpa/hpa_rubyex.yaml:1]: @liangxia  it passed in 4.6, here is the pass log Runner/644512/console

**Pull request merged**